widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -71,6 +73,13 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         if (savedInstanceState == null) {
             startCallWithPermissions();
         }
+
+        // Set title for accessibility screen reader
+        setTitle(
```

Comment:
please refactor this into a separate method to keep `onCreate` cleaner. Method name can replace the need for comments as well.

Reply:
done